### PR TITLE
add images display part 1

### PR DESCRIPTION
**Pull request by thienanh1999**

### CODE REVIEW

File: src/main/java/com/google/codeu/servlets/MessageServlet.java [hunk 1]
```diff
@@ -59,12 +60,28 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
...
+    });
     Gson gson = new Gson();
-    String json = gson.toJson(messages);
+    String json = gson.toJson(messages1);
 
     response.getWriter().println(json);
   }
 
+  /**
+   * Return correct form off image for message
```
Comment (xuanquang1999) [inline src/main/java/com/google/codeu/servlets/MessageServlet.java]: A typo in comment: "off" -> "of"

File: src/main/java/com/google/codeu/servlets/MessageServlet.java [hunk 2]
```diff
@@ -59,12 +60,28 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     }
 
     List<Message> messages = datastore.getMessages(user);
+    List<Message> messages1 = new ArrayList<>();
```
Comment (xuanquang1999) [inline src/main/java/com/google/codeu/servlets/MessageServlet.java]: Can you rename this variable to something more specific than "messages1" (for example, "messagesWithImages")?

**Review by xuanquang1999 — COMMENTED** [review]
LGTM. This is ready to be merged after fixing all the mentioned issues.

File: src/main/java/com/google/codeu/servlets/MessageServlet.java (hunk 1 above)
Comment (thienanh1999) [reply src/main/java/com/google/codeu/servlets/MessageServlet.java]: fixed

File: src/main/java/com/google/codeu/servlets/MessageServlet.java (hunk 2 above)
Comment (thienanh1999) [reply src/main/java/com/google/codeu/servlets/MessageServlet.java]: fixed

**Review by ngthanhtrung23 — APPROVED** [review]

**Pull request merged**